extensions/typescript-language-features/src/test/smoke/referencesCodeLens.test.ts

```diff
@@ -64,7 +64,7 @@ suite('TypeScript References', () => {
 		assert.strictEqual(codeLenses?.[0].range.start.line, 0);
 	});
 
-	test('Should show on basic class properties', async () => {
+	test.skip('Should show on basic class properties', async () => {
```

Comment:
These tests pass locally for me and also seem to pass in CI. If there are causing problems on your machine, let's fix that in a separate PR